airflow/bin/cli.py

```diff
@@ -361,13 +362,87 @@ def run(args, dag=None):
 
     task = dag.get_task(task_id=args.task_id)
     ti = TaskInstance(task, args.execution_date)
-    ti.refresh_from_db()
-    if ti.current_state() == "failed":
-        # This behavior is always correct (failed tasks should not run), but is implemented to specifically handle the
```

Comment:
deleting this comment is really funny

Reply:
It is funny :)

In the interest of being overly clear, this comment remains true - this check is just now a subset of the conditions insisted upon in 367-374